pom.xml

```diff
@@ -71,23 +71,26 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+            </plugin>
             <plugin>
                 <groupId>com.heroku.sdk</groupId>
                 <artifactId>heroku-maven-plugin</artifactId>
                 <version>1.1.1</version>
                 <configuration>
-                    <appName>Back-end</appName>
+                    <appName>back-end</appName>
                     <includeTarget>false</includeTarget>
                     <includes>
-                        <include>Recycle-Buddy/back-end</include>
+                        <include>E:/Masha/RecycleBuddy/back-end/target/recycle-buddy-service-1.0-SNAPSHOT.jar</include>
```

Comment:
this looks like a filepath on your computer, meaning it won't work for anyone else. Maybe a relative filepath would work here?